wp_derive_request_builder/src/generate.rs

```diff
@@ -444,7 +444,7 @@ pub struct ConfigStaticTypes {
 impl ConfigStaticTypes {
     fn new(crate_ident: &Ident) -> Self {
         Self {
-            api_base_url: quote! { std::sync::Arc<#crate_ident::request::endpoint::ApiBaseUrl> },
+            api_base_url: quote! { std::sync::Arc<#crate_ident::ParsedUrl> },
```

Comment:
The generated code still uses `api_base_url` as argument names, which can be fixed by a search-and-replace: `api_base_url` -> `api_root_url`. If the approach is acceptable, I can create a separate PR for that or push another commit.

Reply:
You can push it directly to this PR. Since I already reviewed the PR, I am now catching up on the changes commit by commit. Thanks for checking 👍